e2e/cypress/integration/system_console/reporting/site_statistics_spec.js

```diff
@@ -71,4 +77,131 @@ describe('System Console > Site Statistics', () => {
             cy.wrap(el).eq(0).invoke('text').then(parseFloat).should('be.gt', 0);
         });
     });
+
+    it('MM-T902 - Reporting ➜ Site statistics line graphs show same date', () => {
+        // # Require license.
+        cy.apiRequireLicense();
+
+        goToAdminConsole();
+
+        // * Find site statistics and click it
+        cy.findByTestId('reporting.system_analytics').click();
+
+        let totalPostsDataSet;
+        let totalPostsFromBots;
+        let activeUsersWithPosts;
+
+        // # Grab all data from the 3 charts from there data labels
+        cy.findByTestId('totalPostsLineChart').then((el) => {
+            totalPostsDataSet = el[0].dataset.labels;
+            cy.findByTestId('totalPostsFromBotsLineChart').then((el2) => {
```

Comment:
Currently this test fails for me at this step. Following error is shown:

`Timed out retrying: Unable to find an element by: [data-testid="totalPostsFromBotsLineChart"]`

Reply:
Do you have bot data? It fails if no bot data

Reply:
Do you have suggestions maybe how I can force bot data for this maybe? Just never dealt with them too much so I'm not all that familiar. I'd be open to implement something that fills bot data so this graph is visible.

Reply:
You may refer the test `bot_post_mark_as_unread_spec.js` for steps on how to create bot account and ensure few posts are added by that account. You can also backdate these posts

Reply:
OK Nice thanks Done!